LICENSE.md

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
Yeah I just saw, eh, I really don't like this stuff, two people adding the same file and I don't get a warning.